mirdata/download_utils.py

```diff
@@ -243,13 +244,22 @@ def extractall_unicode(zfile, out_dir):
         out_dir (str): Output folder
 
     """
+    ZIP_FILENAME_UTF8_FLAG = 0x800
```

Comment:
love the refactor! it reads much better now

Reply:
thanks for the review! will merge now